src/main/java/de/adesso/kicker/notification/MatchVerificationRequest/MatchVerificationRequest.java

```diff
@@ -0,0 +1,38 @@
+package de.adesso.kicker.notification.MatchVerificationRequest;
+
+import de.adesso.kicker.match.Match;
+import de.adesso.kicker.notification.Notification;
+import de.adesso.kicker.notification.NotificationType;
+import de.adesso.kicker.user.User;
+import javax.persistence.CascadeType;
+import javax.persistence.ManyToOne;
+
+public class MatchVerificationRequest extends Notification {
+
+    @ManyToOne(targetEntity = Match.class, cascade = CascadeType.ALL)
+    private Match match;
+
+    public MatchVerificationRequest(User sender, User receiver, Match match) {
+
+        super(sender, receiver);
+        super.setType(NotificationType.MATCH_VERIFICATION);
+        this.match = match;
+    }
+
+    public Match getMatch() {
+
+        return match;
+    }
+
+    public void generateMessage() {
+
+        String message = "Bestätige, dass " + getMatch().getWinners().get(0).getFirstName();
+        if (getMatch().getWinners().toArray().length > 1) {
```

Comment:
Why are you converting a List to an Array just to get it's length?
```suggestion
        if (getMatch().getWinners().size() > 1) {
```
Also this is only true if to players are in a team.